uiowa/uiowa
# Block content permissions

Relates to #1890

Sets block content permissions for default sites. Vertical video was purposely omitted because it is my understanding that is homepage only.

## To Test:

- As any webmaster (except policy.clas, hr)  you can't access /admin/structure/block/manage/uids_base_footercontactinfo
- As any webmaster (expect hr) you can edit the Footer Contact Info block but cannot delete. (click the contextual link in the footer)
- As a collegiate or homepage webmaster you can place all the content blocks as normal within the layout builder interface.
- As a webmaster for policy.clas.uiowa.edu there are no new restrictions.
